extensions/gcp/tests/DeleteGCSObjectTests.cpp

```diff
@@ -86,9 +86,7 @@ TEST_F(DeleteGCSObjectTests, ServerGivesPermaError) {
 }
 
 TEST_F(DeleteGCSObjectTests, ServerGivesTransientErrors) {
-  EXPECT_CALL(*delete_gcs_object_->mock_client_, DeleteObject)
-      .WillOnce(testing::Return(TransientError()))
-      .WillOnce(testing::Return(TransientError()));
```

Comment:
Why was this duplicated before, and why can it work with a single `WillOnce` now?

Reply:
It is a change in the patch provided by the google cloud library as mentioned in this comment: https://github.com/googleapis/google-cloud-cpp/discussions/11613#discussioncomment-5973949